src/CLICommand/parameters_helper.ts

```diff
@@ -277,6 +281,37 @@ export class ParametersHelper {
 		return mapFunc(value);
 	}
 
+	// Parse user input through `URL` constructor. This will throw an error on any
```

Comment:
Might be nice to know from the docs that this could throw.

Reply:
Added docblock: [a21f4a1](https://github.com/ardriveapp/ardrive-cli/pull/246/commits/a21f4a1a0cab105eefebb9a6ac63572b730fa5f4)